npm-shrinkwrap.json

```diff
@@ -5,69 +5,70 @@
     "algoliasearch": {
```

Comment:
shrinkwrap madness! I hope npm3 will be ou soon and then Ill make sure everyone upgrades.. in the meantime we will see weird things like that but no big deal